src/box.tsx

```diff
@@ -22,7 +22,16 @@ const Box: BoxComponent = ({ is = 'div', innerRef, children, allowUnsafeHref, ..
    */
   const safeHrefEnabled = (typeof allowUnsafeHref === 'boolean' ? !allowUnsafeHref : getUseSafeHref()) && is === 'a' && parsedProps.href
   if (safeHrefEnabled) {
-    const {safeHref, safeRel} = extractAnchorProps(parsedProps.href, parsedProps.rel)
+    const hrefData:HrefData = {
```

Comment:
Nit: What's the motivation for explicitly marking the type here? What happens if you remove `:HrefData`?

Reply:
If we opt to keep this as-is, I would suggest a space between `hrefData:` and `HrefData`.